src/minecraft-shell.sh

```diff
@@ -93,8 +93,10 @@ case "$CMD" in
 		systemctl --user status "minecraft@$CURRENT_SERVER"
 		;;
 	logs)
-		if [ -z "$CURRENT_SERVER" ]; then
-			echo "No server running."
+		if [ -n "$1" ]; then
+			CURRENT_SERVER=$1
```

Comment:
This breaks `ssh minecraft logs -f`. You should check if `$1` is a positional argument or a switch, i.e. `[[ "$1" != -* ]]` or so. Also, `$1` is not quoted in the assignment.